pkg/receiver/telegrafreceiver/README.md

```diff
@@ -55,3 +55,82 @@ With its current implementation Telegraf receiver has the following limitations:
 - only the following Telegraf metric data types are supported:
   - `telegraf.Gauge` that is translated to `pdata.MetricDataTypeGauge`,
   - `telegraf.Counter` that is translated to `pdata.MetricDataTypeSum`.
+
+## Migration from Telegraf
+
+### Data model
+
+Internal OTC metric format differs from the Telegraf one and `separate_field` controls the conversion:
+
+- If `separate_field` is `false`, the Open Telemetry metric name is going to be concatenated from the Telegraf metric name
+  and the Telegraf field with `_` as separator.
+- If `separate_fields` is `true`, the Open Telemetry metric name is going to be the same as the Telegraf one,
+  and the Telegraf `field` is going to be converted to the Open Telemetry data point attribute.
+
```

Comment:
Would be nice to show an example here. For people unfamiliar with Telegraf's data model, this isn't going to be very informative on its own.